src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -140,6 +144,24 @@ export class WorkspaceSavedObjectsClientWrapper {
     return false;
   };
 
+  private isDashboardAdmin(request: OpenSearchDashboardsRequest): boolean {
```

Comment:
Prefer renaming to `isRequestByDashboardAdmin`.

Reply:
Done